drivers/util/util.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// SplitPortProto splits a string in the format port/protocol, defaulting
+// protocol to "tcp" if not provided.
+func SplitPortProto(raw string) (port int, protocol string, err error) {
```

Comment:
I'd say per Go way of handling things, let it parse port number as `string`. https://golang.org/pkg/net/#SplitHostPort